src/js/index.js

```diff
@@ -248,16 +248,59 @@ export const hpe = deepFreeze({
     },
   },
   checkBox: {
-    color: 'selected-text',
-    gap: 'small',
-    check: {
-      radius: '2px',
+    hover: {
+      border: {
+        color: undefined,
+      },
+      background: {
+        color: 'background-contrast',
+      },
     },
     border: {
+      color: 'border',
       width: '1px',
     },
+    check: {
+      radius: '2px',
+      extend: ({ theme, checked, indeterminate }) => `
+      background-color: ${
+        checked || indeterminate
+          ? theme.global.colors['green!']
+          : theme.global.colors.background[theme.dark ? 'dark' : 'light']
+      };
+      ${(checked || indeterminate) && 'border: none;'}
+        `,
+    },
+    icon: {
+      extend: 'stroke-width: 2px;',
+    },
+    gap: 'small',
+    toggle: {
+      background: 'background',
+      color: 'background',
+      knob: {
+        extend: ({ theme }) => `
+           box-shadow: ${
+              theme.global.elevation[theme.dark ? 'dark' : 'light'].small
+            };
+           border: 1px solid ${
+             theme.global.colors.border[theme.dark ? 'dark' : 'light']
+           }
+        `,
+      },
+      extend: ({ checked, theme }) => `
+        ${checked && `background-color: ${theme.global.colors['green!']};`}
+      `,
+    },
+    extend: ({ theme }) => `
+      width: 100%;
+      padding: ${theme.global.edgeSize.small};
```

Comment:
Confirming that this adds back in the pad that is removed in 301? and by doing this, the hover background fills the entire perceived formfield background @halocline

Reply:
Correct. Removing content.pad from FormField which is only applied to CheckBox, RadioButtonGroup, RangeInput. Instead, adding pad to CheckBox for background fills.

Relevant Grommet:
https://github.com/grommet/grommet/blob/2eb8d09df21c2e6bfc86bd069e5bfaa1e9193ab9/src/js/components/FormField/FormField.js#L21
https://github.com/grommet/grommet/blob/2eb8d09df21c2e6bfc86bd069e5bfaa1e9193ab9/src/js/components/FormField/FormField.js#L174

Reply:
This was the direction we decided on. 👍